packages/run-it/src/components/CopyClipboardIconButton/CopyClipboardIconButton.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import React, { FC, useRef, useState } from 'react'
+import { IconButton } from '@looker/components'
+import { Done } from '@styled-icons/material/Done'
+import { Assignment } from '@styled-icons/material/Assignment'
+
+export interface CopyClipboardIconButtonProps {
+  content: string
+  tooltip?: string
+  success?: string
+}
+
+export const CopyClipboardIconButton: FC<CopyClipboardIconButtonProps> = ({
+  content,
+  tooltip = 'Copy to clipboard',
+  success = 'Copied',
+}) => {
+  const [copied, setCopied] = useState(false)
+  const buttonRef = useRef<HTMLButtonElement>(null)
+
+  const clickCopyButton = () => {
+    const textField = document.createElement('textarea')
```

Comment:
The element in the `if buttonRef.current` block. This has the potential to leak although in theory it wont as buttonRef current should never be null at this point.

Reply:
I noticed that too. This is copied from the `CopyToClipboard` component in `@looker/components` and I just wanted to get the icon version working correctly first before I changed the code more.